mousestyles/data/__init__.py

```diff
@@ -62,3 +62,80 @@ def load_all_features():
     other_features = [x.iloc[:, -1] for x in data_frames]
     other_features.insert(0, all_features_df)
     return pd.concat(other_features, axis=1)
+
+
+def load_intervals(feature):
+    """
+    Parameters
+    ----------
+    feature: {"AS", "F", "IS", "M_AS", "M_IS", "W"}
+
+    Returns
+    -------
+    intervals : pandas.DataFrame
+        All the data from the feature directory into one data frame
+        columns are strain, mouse, day, start, stop
+    """
+    # get all file names
```

Comment:
@changsiyao - perhaps a new separate function just for input validation e.g. `test_feature_load_input` and add multiple asserts there. This will modularise the type of tests out more